kiwi/utils/rpm.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2019 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+import os
+import re
+
+# project
+from kiwi.command import Command
+from kiwi.defaults import Defaults
+from kiwi.path import Path
+
+
+class Rpm(object):
+    """
+    **Helper methods to handle the rpm database configuration**
+    """
+    def __init__(self, root_dir=None):
+        self.root_dir = root_dir
+        self.config = []
+        self.custom_config = []
+        self.macro_path = os.sep.join(
+            [
+                self.root_dir or '', Defaults.get_custom_rpmdb_macros_path()
+            ]
+        )
+        self.macro_file = os.sep.join(
+            [self.macro_path, 'macros.kiwi']
```

Comment:
I'd probably make this filename more specific: `macros.kiwi-rpmdb-cfg` or something. Just so that it's clear what it's for.

Reply:
we will use this macro file also for fixing the lang issue. So I suggest to be a bit more generic. How about:

* macros.kiwi-bootstrap-config

Thoughts ?

Reply:
I prefer generic also because this can be later used for further purposes.

Also thinking more about this macro feature I believe it could be an interesting feature to allow some customization over it, I mean let user append some stuff in it. The fact that this enables setting custom macros for bootstrap is valuable. I struggled with it while trying to slim down container images using tricks similar to the lang issue (I used some macros to no install in `/usr/share/doc` and similar artifacts). By then I'd love to have a way to customize macros during bootstrap.

Anyway the point is that I think this macro strategy might have further use cases in the future, so I would not limit it to the db path issue.

Reply:
@Conan-Kudo do you agree on the proposed name ?

Reply:
The install-langs macro may actually probably stick around post-build so that it stays in effect.